crossbeam-skiplist/src/set.rs

```diff
@@ -455,7 +454,7 @@ impl<'a, T> Entry<'a, T> {
     }
 
     /// Returns a reference to the value.
-    pub fn value(&self) -> &'a T {
```

Comment:
This appears to have been accidentally changed.

Reply:
Ah, sorry